src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -84,7 +85,8 @@ const DeleteOrgButton: FC = () => {
     hidePopup()
   }
 
-  const shouldShowUsersWarning = users.length > 1
+  // only show warning for free accounts with multiple users
+  const shouldShowUsersWarning = account.type === 'free' && users.length > 1
 
```

Comment:
The meaning of this term has changed because there's more than one warning now - you get a warning of one type with multiple users in a free account, and a warning once you make it into the overlay with multiple users in a PAYG or Contract account.

How about `freeAccountWithOtherUsers`?